rules/rbe_repo.bzl

```diff
@@ -134,60 +262,97 @@ There are two modes of using this repo rules:
                 --platforms=@rbe_default//config:platform \
                 --extra_toolchains=@rbe_default//config:cc-toolchain \
 
-    Note running bazel clean --expunge_async, or otherwise modifying attrs or
-    env variables used by this rule will trigger it to re-execute. Running this
-    repo rule can take some time (if you are not using the rbe-ubuntu 16_04 container)
-    as it needs to pull a container, run it, and then run some commands inside.
-    We recommend you use output_base and check in the produced
-    files so you dont need to run this rule with every clean build.
+NOTE 2: WHEN DOES THIS RULE PULL A CONTAINER
 
-The {bazel_version} above corresponds to the version of bazel installed locally.
-Note you can override this version and pass an optional rc # if desired.
-Running this rule with a non release version (e.g., built from source) will result in
-picking as bazel version _BAZEL_VERSION_FALLBACK. Note the bazel_version / bazel_rc_version
-must be published in https://releases.bazel.build/...
+Most users of rbe_autoconfig do not expect their target to pull a container.
+If your rbe_autoconfig rule nevertheless pulls a container its because it
+could not find checked-in configs that match:
+1- The Bazel version you are currently using
+2- The container you selected (if you are setting also base_container_digest)
```

Comment:
> if you are setting also base_container_digest

This doesn't quite make sense to me. Did you mean `if you are also setting ...` or something else?

Reply:
thanks, fixed